main.go

```diff
@@ -75,5 +90,8 @@ func main() {
 
 func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
-	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+
+	flag.StringVar(&coreEtcdURL, "etcd-core-url", "", "The address of the core etcd server. Required.")
```

Comment:
for the cobra follow-up: use the CoreEtcdOptions directly here and follow the patterns we use in kube for this: https://github.com/sttts/kubernetes/blob/956bbfd1a6764e6bb17e756ffb5065c5b385fb95/cmd/kube-controller-manager/app/options/options.go

Reply:
That's the plan, and I'm already doing that for flags in #15. I just need to improve how the struct is passed between main and EtcdProxyController.